client/src/App.js

```diff
@@ -2,17 +2,33 @@ import React from "react";
 import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route } from "react-router-dom";
 
-import { theme } from "./themes/theme";
-import LandingPage from "./pages/Landing";
+// User Context Information
+import { UserContext } from "./components/User/UserContext";
+import getUserData from "./components/User/getUserData";
```

Comment:
For clarity, it would be best to keep this name consistent with the name in the file `useFindUser`

Reply:
Oops.  Remnant of changing the name.  getUserData was not recognized as a valid hook anyway, so it has been renamed in all places to "useGetUserData".  Push will come shortly.